usagov/usagov-2021
# USAGOV-1912: sub page breadcrumb spacing update

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1912

## Description

Update spacing above and below the breadcrumb on sub pages
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps
This is a visual test to make sure the spacing above and below the breadcrumb looks the same on desktop, tablet and mobile.
Goto http://localhost/facts-figures on desktop. Look at the spacing above and below the breadcrumb and check the page in tablet and mobile to see if the spacing looks the same.

<img width="1512" alt="Screenshot 2024-09-04 at 7 23 56 AM" src="https://github.com/user-attachments/assets/ac0d4444-4de6-47f1-9162-ad3c712f5654">

<img width="1512" alt="Screenshot 2024-09-04 at 7 24 30 AM" src="https://github.com/user-attachments/assets/75ff5e15-e144-4bbb-af03-dd0c0240fa74">


- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
